apps/src/lab2/hooks/useFileUploader.tsx

```
@@ -63,6 +72,26 @@ const isValidMimeType = (
   );
 };
 
+const moderateImage = async (
+  file: File,
+  ext: string,
+  appName?: string
+): Promise<'ok' | 'flagged' | 'skipped'> => {
+  if (appName !== 'weblab2' || !['png', 'jpg', 'jpeg'].includes(ext)) {
+    return 'skipped';
+  }
+  const response = await HttpClient.post(`/v3/images/moderate`, file, true, {
+    'Content-Type': file.type || 'application/octet-stream',
+  });
+  if (!response.ok) {
+    throw new Error('Error with image moderation.');
```

should we throw an error here or silently move on? Would we want to crash the page if there was an issue with image moderation?

---

Yes, makes sense to not block user here!
Will log error and return 'skipped'.